dashboard_description_plugin/package.json

```diff
@@ -1,123 +1,130 @@
 {
     "name": "dashboard-description-plugin",
-    "version": "1.1.0",
+    "version": "1.1.1",
```

Comment:
@dremileta here we list the V2 changes https://github.com/gooddata/gooddata-dashboard-plugins/pull/48/files#diff-7da791616f8d20af898514df1083e884eaa5a57f7f0c3a29389699fecbc19172R15, since there's been a breaking change in the wrap property of the plugin config, please use the same version as in the internal repo.